### PR TITLE
Fix test_get_pipeline_steps for python 3.5

### LINKED ISSUE #398: test_get_pipeline_steps failing on 3.5 Jenkins
Seems as though the new Python 3.5 Jenkins build added in #387 doesn't quite succeed.

The assertion in `test_get_pipeline_steps()` in `test_pipeline_tools.py` is failing:
https://ssbjenkins.stsci.edu/blue/organizations/jenkins/STScI%2Fjwql/detail/develop/17/pipeline/22

I assume (hope!) this is a simple version issue.

(Also, why isn't this failing the Jenkins build??? The question that never goes away...)

### PR DESCRIPTION
**Pull request by bourque**
This PR modifies `test_pipeline_tools` in order to get the tests within to pass in a Python 3.5 environment.

Closes #398

### CODE REVIEW

**Comment by bourque** [comment]
@bhilbert4 I think this fixes #398, however I am uncertain if asserting that the contents of the pipeline dictionaries are the same regardless of order is ok to do.  Could you comment/review?

**Comment by bourque** [comment]
I updated the tests to test only truly required steps, in order, and the tests appear to be passing! I will go ahead and merge this.

File: jwql/tests/test_pipeline_tools.py [hunk 1]
```diff
@@ -56,29 +56,32 @@ def test_completed_pipeline_steps():
                                   ('jump', True),
                                   ('rate', True)])
 
-    assert completed_steps == true_completed
+    assert dict(completed_steps) == dict(true_completed)
```
Comment (laurenmarietta) [inline jwql/tests/test_pipeline_tools.py]: Just want to double-check - @bourque did you mean for the `test_completed_pipeline_steps()` assertion to also be modified in this PR?

Comment (bourque) [reply jwql/tests/test_pipeline_tools.py]: @laurenmarietta Whoops, nice catch!  I'll make the update.

**Pull request merged**